cmd/misc/main.go

```diff
@@ -260,6 +260,19 @@ func ClearBigtable(family string, key string, dryRun bool, bt *db.Bigtable) {
 }
 
 func IndexOldEth1Blocks(startBlock uint64, endBlock uint64, batchSize uint64, concurrency uint64, transformerFlag string, bt *db.Bigtable) {
+	if endBlock > 0 && endBlock < startBlock {
```

Comment:
I don't understand `endBlock > 0` here. If `endBlock` is 0 and `startBlock` is 1 then this check does not trigger an error even though it should.
Shouldn't it be possible to simply remove the `endBlock > 0`? Why is it needed?

Reply:
because if endBlock is 0 then we we run from the startBlock to the current one.